pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdRead.java

```diff
@@ -101,6 +102,9 @@ public class CmdRead extends AbstractCmdConsume {
     @Option(names = { "-pm", "--pool-messages" }, description = "Use the pooled message", arity = "1")
     private boolean poolMessages = true;
 
+    @Option(names = { "-mp", "--print-metadata" }, description = "Message metadata")
```

Comment:
`-mp` isn't very intuitive. perhaps there doesn't need to be a short option and `--print-metadata` would be sufficient? (no strong opinion on this)

Reply:
long option is mostly useful for documentation but the short option is useful when any user would like to use such tool on-demand and it can be used without dealing with typos. also, for consistency with other params that have a combination of short and long options, maybe it's better to keep both options for this new param as well.